logsmanagement/logsmanagement.conf

```diff
@@ -0,0 +1,99 @@
+[System Logs]
+	enabled = yes
+	update every = 1
+	log type = flb_systemd
+	circular buffer max size = 256 # in MiB
+	compression acceleration = 1 # see https://github.com/lz4/lz4/blob/90d68e37093d815e7ea06b0ee3c168cccffc84b8/lib/lz4.h#L195
+	buffer flush to DB = 6 # in sec, default 6 min 4
+	disk space limit = 500 # in MiB, default 500MiB
+	priority value chart = yes
+	severity chart = yes
+	facility chart = yes
+
+[Docker Events Logs]
+	enabled = yes
+	update every = 1
+	log type = flb_docker_events
+	circular buffer max size = 256 # in MiB
+	compression acceleration = 1 # see https://github.com/lz4/lz4/blob/90d68e37093d815e7ea06b0ee3c168cccffc84b8/lib/lz4.h#L195
+	buffer flush to DB = 6 # in sec, default 6 min 4
+	disk space limit = 500 # in MiB, default 500MiB
+	event type chart = yes
+
+[Apache access.log]
```

Comment:
Apache has two default files, I suggest to include `error.log` too.

Reply:
Hmm OK, but I am not sure how to parse that yet (what information do we want to extract from `error.log` ?). Same goes for any `error.log` files really, I need to have a think what we want to monitor.

Reply:
Number of wrong [acesses](https://www.a2hosting.com/kb/developer-corner/apache-web-server/viewing-apache-log-files)  can indicate possible DoS attack or either bugs on sites, this is the motive I think we could also parse it. :thinking:

Reply:
OK, maybe I can set up a custom chart based on a regex that checks for this, to be enabled by default in the configuration. Just as an example and then the users can extend it as they wish.